tools/samtools.wdl

```diff
@@ -1327,7 +1327,90 @@ task faidx {
         cpu: 1
         memory: "4 GB"
         disks: "~{disk_size_gb} GB"
-        container: "quay.io/biocontainers/samtools:1.17--h00cdaf9_0"
+        container: "quay.io/biocontainers/samtools:1.19.2--h50ea8bc_0"
+        maxRetries: 1
+    }
+}
+
+task sort {
+    meta {
+        description: "Sorts the input BAM file"
+        external_help: "https://www.htslib.org/doc/samtools-sort.html"
+        outputs: {
+            sorted_bam: "The input BAM after it has been sorted",
+            sorted_bam_index: "The `.bai` BAM index file associated with `sorted_bam`",
+        }
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file to sort"
+        tag_sort: {
+            description: "Sort by the value of the alignment tag specified in `tag_sort`. When specified, samtools sorts by the value of the alignment tag and then by either position or read name.",
+            external_help: "https://www.htslib.org/doc/samtools-sort.html",
+        }
+        prefix: "Prefix for the output file. The extension `.bam` will be added."
+        uncompressed: "Output uncompressed BAM?"
```

Comment:
I think until streaming comes to WDL, we should probably avoid allowing uncompressed output

Reply:
Some engines (e.g. dxCompiler) support streaming at the moment.